### PR TITLE
Change from last review forward

### PR DESCRIPTION
**Pull request by ThisisYoYoDev**
## Summary

Some frontend change and new prisma schema for question and answer

## Checklist

**Bureaucracy**

- [ ] I set a descriptive title
- [x] I set a proper description, providing enough information to the reviewer
- [x] I considered squashing this PR, if needed
- [ ] I set the proper assignee
- [ ] I set the proper reviewers

**Quality Assurance**

- [ ] I tested the behavior of this PR locally and made sure it complies with
  the ticket description

### CODE REVIEW

File: backend/prisma/schema.prisma [hunk 1]
```diff
@@ -114,11 +114,37 @@ model Lesson {
...
-  Lesson      Lesson @relation(references: [id], fields: [lesson_id], onDelete: Cascade)
-  lesson_id   String @db.Uuid
-  title       String
-  description String?
-  data        String
+  id              String       @id @default(uuid()) @db.Uuid
+  Lesson          Lesson       @relation(references: [id], fields: [lesson_id], onDelete: Cascade)
+  lesson_id       String       @db.Uuid
+  title           String
+  type_answer     AnswerType
```
Comment (alexandregrare) [inline backend/prisma/schema.prisma:136]: answer_type

File: backend/prisma/schema.prisma [hunk 2]
```diff
@@ -114,11 +114,37 @@ model Lesson {
...
+  type_question   QuestionType
```
Comment (alexandregrare) [inline backend/prisma/schema.prisma:137]: question_type

**Review by Toumi-Elyes — APPROVED** [review]

File: backend/src/question/question.dto.ts [hunk 3]
```diff
@@ -40,6 +41,14 @@ export class CreateQuestionModel {
   @ApiProperty()
   @IsString()
   Data: string;
+
+  @ApiProperty()
+  @IsString()
+  TypeAnswer: AnswerType;
```
Comment (alexandregrare) [inline backend/src/question/question.dto.ts:47]: lowercase

File: backend/src/question/question.service.ts [hunk 4]
```diff
@@ -23,6 +23,8 @@ export class QuestionService {
           title: questionData.Title,
           description: questionData.Description,
           data: questionData.Data,
+          type_answer: questionData.TypeAnswer,
```
Comment (alexandregrare) [inline backend/src/question/question.service.ts:26]: answer_type

File: backend/src/question/question.service.ts [hunk 5]
```diff
@@ -23,6 +23,8 @@ export class QuestionService {
...
+          type_question: questionData.TypeQuestion,
```
Comment (alexandregrare) [inline backend/src/question/question.service.ts:27]: question_type

File: backend/src/question/question.dto.ts [hunk 6]
```diff
@@ -69,3 +78,18 @@ export class UpdateQuestionModel {
   @IsOptional()
   Data: string;
 }
+
+export class CreateAnswerModel {
+  @ApiProperty()
+  @IsString()
+  Value: string;
```
Comment (alexandregrare) [inline backend/src/question/question.dto.ts:85]: lowercase

File: backend/src/question/question.dto.ts [hunk 7]
```diff
@@ -69,3 +78,18 @@ export class UpdateQuestionModel {
...
+
+  @ApiProperty()
+  @IsUUID()
+  QuestionId: string;
```
Comment (alexandregrare) [inline backend/src/question/question.dto.ts:89]: lowercase

File: backend/src/question/question.dto.ts [hunk 8]
```diff
@@ -69,3 +78,18 @@ export class UpdateQuestionModel {
...
+
+  @ApiProperty()
+  @IsNumber()
+  @IsOptional()
+  Point: number;
```
Comment (alexandregrare) [inline backend/src/question/question.dto.ts:94]: lowercase

File: frontend/src/Components/Button/ArrowButton.tsx [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+interface ArrowButtonProps {
+  onClick: () => void
+  buttonText: string
+  styleColor?: StyleColor;
+};
+
+const ArrowButton = ({ onClick, buttonText, styleColor}: ArrowButtonProps) : JSX.Element => {
+  return (
+    <Container style={colorStyles[styleColor ?? 'WHITE']} onClick={onClick}>
```
Comment (alexandregrare) [inline frontend/src/Components/Button/ArrowButton.tsx:18]: StyleColor.White

**Review by alexandregrare — CHANGES_REQUESTED** [review]

**Pull request merged**